### DISCUSSION TITLE
New models have less MTBF?

### DISCUSSION THREAD

**Discussion by sunCelery** [linuxhw/SMART#3] [Q&A] [answered]
It seems what I've notice one interesting thing the newer models of drives have lower MTBF is it because of they have not been worked enough yet? Or they just worse? 

E.G. WD two red class drives:
WD20EFRX ~1.75 (the older one ~2012)
WD20EFAX = 0,11 (newer one ~2019)

P.S.:
The project is awesome and very informative 
Thanks for it

**Comment by KOLANICH** [comment, marked as answer]
I have done a reserch on it. The short answer is `no`, the year of first occurrence of a model is not a risk factor. At least neuther KM-curves, nor Cox PH, nor Weibull model show this. The main risk factor is brand. Everything else is very insignificant and even more advanced models (XGBoost and neural networks) have failed to capture the interactions. I gonna do an additional research on it, but it becomes a bit lengthy.

**Reply by sunCelery** [reply]
Wow, the answer is bigger than awaited, thanks a lot.
I would like to read the results of this further research)

**Reply by Gomesz785** [reply]
One of my issues as well, good answer you have written here.
